sbt-test/src/main/scala/hedgehog/sbt/Framework.scala

```diff
@@ -98,14 +100,28 @@ case class Event(
 object Event {
 
   def fromReport(taskDef: sbtt.TaskDef, selector: sbtt.Selector, report: Report, duration: Long): Event = {
-    val status = report.status match {
-      case Failed(_, _) =>
-        sbtt.Status.Failure
+    val (status, maybeThrowable) = report.status match {
+      case Failed(_, log) =>
+        (sbtt.Status.Failure, Some[Throwable](new StackTraceMutedException(log.map(Test.renderLog).mkString("\n"))))
       case GaveUp =>
-        sbtt.Status.Error
+        (sbtt.Status.Error, None)
       case OK =>
-        sbtt.Status.Success
+        (sbtt.Status.Success, None)
     }
-    Event(taskDef.fullyQualifiedName, taskDef.fingerprint, selector, status, None, duration)
+    Event(taskDef.fullyQualifiedName, taskDef.fingerprint, selector, status, maybeThrowable, duration)
   }
 }
+
+/** 
+  * This exception ignores printStackTrace with the given PrintStream or PrintWriter argument
+  * in order to avoid printing noisy and useless stacktrace
```

Comment:
I would necessarily say "useless" here. It's more "misleading" because the stacktrace is coming from here and no the assertion. :)

Reply:
Ah yeah you're right. I'll fix it.